PlaidLink/Classes/PLDLinkNavigationViewController.h

```diff
@@ -71,7 +71,7 @@
 @property(nonatomic, readonly) PlaidProduct product;
 
 // Optional Properties
-@property(nonatomic, copy) NSString *webhook;
+@property(nonatomic, weak) NSMutableDictionary *options;
```

Comment:
Please make the an NSDictionary, the assumption being that it will not be modified